src/dmd/cppmangle.d

```diff
@@ -510,10 +513,145 @@ private final class CppMangleVisitor : Visitor
                 append(ti.tempdecl);
                 this.writeIdentifier(ti.tempdecl.toAlias().ident);
             }
+            abi_tags(abiTags);
             template_args(ti);
         }
         else
+        {
             this.writeIdentifier(s.ident);
+            abi_tags(abiTags, s.isNspace() !is null);
+        }
+    }
+
+    /******************************
+     * Write the mangled representation of the abi-tags.
+     * Params:
+     *  tags = slice of tags
+     *  isnspace = `true` if the current symbol is a namespace
+     */
+    void abi_tags(const(char)*[] tags, bool isnspace = false)
```

Comment:
Please use D naming conventions:

* `abi_tags` -> `abiTags`
* `isnspace` -> `isNamespace`

Reply:
This is inline with the convention of the surrounding code.

Reply:
We want to improve the existing code and new code should follow the same improvements. We don't want to move backwards.

Reply:
Well, consistently ugly is better than Frankenstein.

Reply:
@SSoulaimane Please follow the [D Style](https://dlang.org/dstyle.html) for all new code unless you're explicitly asked not to.

Reply:
Changed.